homeassistant/components/alexa/__init__.py

```diff
@@ -31,7 +48,8 @@
                 vol.Required(CONF_TEXT, default=""): cv.template,
                 vol.Optional(CONF_DISPLAY_URL): cv.template,
             }]),
-        }
+        },
+        CONF_SMART_HOME: vol.Any(SMART_HOME_SCHEMA, None),
```

Comment:
Instead of `vol.Any`, just mark the key as `vol.Optional` ([like this](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/cloud/__init__.py#L83))

Reply:
You probably have to wrap the containing dictionary in a `vol.Schema`: `DOMAIN: vol.Schema({`

Reply:
Oh I guess you still need to use Any with None since the presence of the key indicates setting up the smart home view.

Nevermind this comment.